public/configurations/visualizations/aar-flow-sla-heatmap.json

```diff
@@ -24,5 +24,21 @@
             { "column": "slastatus", "label": "Status" }
         ]
     },
+    "filterOptions": {
```

Comment:
@nxanil this snippet needs to change to something like:

"filterOptions": {{ nsgListFilterOptions }}

Where nsgListFilterOptions is a variable which is passed around in context.

Psudocode could be like:
'''
nsgListFilterOptions = {"Destination GW": { "parameter": "dnsg"}}
for nsg in nsgList: {
    nsgListFilterOptions['Destination GW']['options'].append({ 'label':  nsg, 'value': nsg})
}
nsgListFilterOptions['Destination GW']['default'] = nsg
'''

Thus you create nsgListFilterOptions dynamically. @sayaji15  has mentioned where we populate NSG list in our framework. so you can create this variable there. Also, it is imp to make sure that it changes whenever that list is modified. 

Hope that clears.